async.md

```diff
@@ -218,3 +220,100 @@ Example
 			resp.error("caught: "+reason.message);
 		})
 ~~~
+
+# Messaging
+
+This class allows for the execution of async ClearBlade messaging functions provide an MQTT client that will work asynchronously. You can connect to the internal ClearBlade broker or an external broker. 
```

Comment:
typo? "This class allows for the execution of async ClearBlade messaging functions provide an MQTT client that will work asynchronously" doesn't make sense to me.